src/main/java/seedu/address/logic/commands/DeleteMeetingCommand.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Meeting;
+import seedu.address.model.person.Name;
+import seedu.address.model.person.Person;
+
+/**
+ * Deletes a meeting from an Udder's list of meetings.
+ */
+public class DeleteMeetingCommand extends Command {
+
+    public static final String COMMAND_WORD = "deletem";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the meeting identified by the index number used in the command result panel.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    public static final String MESSAGE_DELETE_MEETING_SUCCESS = "Deleted Meeting: %1$s";
+
+    private final Index targetIndex;
+
+    public DeleteMeetingCommand(Index targetIndex) {
+        this.targetIndex = targetIndex;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+
+        if (targetIndex.getZeroBased() >= model.getMeetingSize()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_MEETING_INDEX);
+        }
+
+
+        Meeting toDelete = model.getMeeting(targetIndex.getZeroBased());
+        Name target = toDelete.getPersonToMeet();
+        Person personToDeleteMeeting = null;
+        for (Person person : model.getFilteredPersonList()) {
+            if (person.getName().equals(target)) {
+                personToDeleteMeeting = person;
```

Comment:
There might be a bug where the name of the person might be changed using Edit Command and not changed in the meetings, leaving meetings with invalid names from the list! Unless maybe you have considered this somewhere?

Reply:
Thanks for pointing that out. I'll work on that in for `v1.5`.